docs/source/getting_started/dataset_creation/dataset_creation_gcloud_bucket.rst

```diff
@@ -1,6 +1,164 @@
+- Add a condition with the title `BUCKET_PROJECTS_WILD_ANIMALS`
+  and insert the condition below in the Condition editor. Remember to change the bucket name
+  and path to the folder. However, you must keep the "objects" inbetween.
+  For more information, head to the `IAM conditions
+  <https://cloud.google.com/storage/docs/access-control/iam#conditions>`_.
+  The first part of the condition adds listing rights to the whole bucket,
+  as they can only be handled on the bucket level. The second part adds object-level
+  access rights (i.e. read and create) for all objects in the bucket `lightly-datalake`
+  whose name starts with `projects/wild-animals`.
+
+.. code::
+
+    (
+        resource.type == 'storage.googleapis.com/Bucket' &&
+        resource.name.startsWith("projects/_/buckets/lightly-datalake")
+    ) || (
+        resource.type == 'storage.googleapis.com/Object' &&
+        resource.name.startsWith("projects/_/buckets/lightly-datalake/objects/projects/wild-animals")
+    )
+
+.. figure:: images_gcloud_bucket/screenshot_gcloud_create_service.jpg
+    :align: center
+    :alt: Google Cloud Service Account
+    :width: 60%
+
+- Click on `Done` to create the service account.
+- You can change the roles of the service account later in the
+  `IAM <https://console.cloud.google.com/iam-admin/iam>`_.
+
+6. Navigate to `APIs -> Credentials <https://console.cloud.google.com/apis/credentials>`_
+   again if you are not already there.
+
+- Find the just created user in the list of all service accounts.
+- Click on the user and navigate to the `keys` tab.
+- Click on `Add key` and create a new private key in JSON Format.
+  It will download the corresponding key file.
+
+.. figure:: images_gcloud_bucket/screenshot_gcloud_service_account_key_creation.jpg
+    :align: center
+    :alt: Google Cloud Service Account Key Creation
+    :width: 60%
+
+
+
+
+Configuring a Lightly dataset to access the Google Cloud Storage
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Create and configure a dataset
+
+1. `Create a new dataset <https://app.lightly.ai/dataset/create>`_ in Lightly
+2. Edit your dataset and select `Google Cloud Storage` as your datasource
+
+.. figure:: images_gcloud_bucket/screenshot_gcloud_create_dataset.jpg
+    :align: center
+    :alt: Configure google cloud bucket datasource in Lightly Webapp
+    :width: 60%
+
+
+3. As the resource path, enter the full URI to your resource eg. `gs://lightly-datalake/projects/wild-animals`
+4. Enter the Google Project ID you wrote down in the first step.
+5. Click on `Select Credentials File` to add the key file you downloaded in the previous step.
+6. The thumbnail suffix allows you to configure
+
+- where your thumbnails are stored when you already have generated thumbnails in your S3 bucket
+- where your thumbnails will be stored when you want Lightly to create thumbnails for you.
+  For this to work, the user policy you created must be granted write permissions.
```

Comment:
maybe note here that using `uploads=thumbnail` would be required?

Reply:
I added that in the last chapter about `Create the dataset and upload embeddings and metadata`, as it is used in that step.